hadoop-ozone/dist/src/shell/ozone/ozone-functions.sh

```diff
@@ -2661,6 +2662,110 @@ function ozone_generic_java_subcmd_handler
   fi
 }
 
+## @description Validates if all jars as indicated in the corresponding ${OZONE_RUN_ARTIFACT_NAME} classpath file are present
+## @audience private
+## @stability evolving
+## @replaceable yes
+function validate_classpath
+{
+  if [ -n "$1" ]; then
+    case "$1" in
+      -h|--help)
+        echo "Usage I: ozone classpath <ARTIFACTNAME> --validate"
+        echo "Usage II: ozone --daemon start|status|stop csi|datanode|om|recon|s3g|scm --validate"
+        echo
+        echo "Options:"
+        echo
+        echo "  -h, --help"
+        echo "      This help text."
+        echo
+        echo "  --validate"
+        echo "      Validates if all jars as indicated in the corresponding OZONE_RUN_ARTIFACT_NAME classpath file are present."
+        echo
+        exit 0
+        ;;
+      --validate)
+        validate_classpath_util "$2"
+        ;;
+      *)
+        echo "Invalid option "$1". Use --help to see the valid options."
+        exit 1
+        ;;
+    esac
+  fi
+}
```

Comment:
This way of parsing arguments is very fragile.  Please add the new option in `ozone_parse_args` instead.

Reply:
Thank you @adoroszlai for the review, incorporated the suggested changes!